backend/sokdak/src/test/java/com/wooteco/sokdak/notification/service/NotificationServiceTest.java

```diff
@@ -156,4 +161,43 @@ void existsNewNotification(Long memberId, boolean expected) {
 
         assertThat(newNotificationCheckResponse.isExistence()).isEqualTo(expected);
     }
+
+    @DisplayName("알림을 반환한다.")
+    @Test
+    void findNotifications() {
+        Notification notification = Notification.builder()
+                .member(member)
+                .notificationType(HOT_BOARD)
+                .post(post)
+                .build();
+        Notification notification2 = Notification.builder()
+                .member(member)
+                .notificationType(POST_REPORT)
+                .post(post)
+                .build();
+        Notification notification3 = Notification.builder()
+                .member(member)
+                .notificationType(NEW_COMMENT)
+                .post(post)
+                .comment(comment)
+                .build();
+        notificationRepository.save(notification);
+        notificationRepository.save(notification2);
+        notificationRepository.save(notification3);
+
+        NotificationsResponse notificationsResponse = notificationService
+                .findNotifications(AUTH_INFO, PageRequest.of(0, 2, Sort.by("createdAt").descending()));
+        List<NotificationResponse> notificationResponses = notificationsResponse.getNotifications();
+
+        assertAll(
+                () -> assertThat(notificationsResponse.isLastPage()).isFalse(),
+                () -> assertThat(notificationResponses).hasSize(2),
+                () -> assertThat(notificationResponses.get(0).getPostId()).isEqualTo(post.getId()),
+                () -> assertThat(notificationResponses.get(0).getType()).isEqualTo(NEW_COMMENT),
+                () -> assertThat(notificationResponses.get(0).getContent()).isEqualTo(comment.getMessage()),
+                () -> assertThat(notificationResponses.get(1).getPostId()).isEqualTo(post.getId()),
+                () -> assertThat(notificationResponses.get(1).getType()).isEqualTo(POST_REPORT),
+                () -> assertThat(notificationResponses.get(1).getContent()).isEqualTo(post.getTitle())
```

Comment:
이건 그냥 제안인데, 해당 알림 목록에 대해 postId, type, content를 검증하는 메소드를 추출해도 좋을 것 같아요.
```suggestion
               validateNotificationResponse(notificationResponses.get(0), post.getId(), NEW_COMMENT, comment.getMessage());
              validateNotificationResponse(notificationResponses.get(1), post.getId(), POST_REPORT, post.getTitle());
              private void validateNotificationResponse(NotificationResponse notificationResponse, Long postId, NotificationType type, String message) {
              assertAll(
                      () -> assertThat(notificationResponse.getPostId()).isEqualTo(postId),
                      () -> assertThat(notificationResponse.getType()).isEqualTo(type.toString()),
                      () -> assertThat(notificationResponse.getContent()).isEqualTo(message)
              );
          }
        
```